packages/gestalt/src/ButtonToggle.tsx

```diff
@@ -186,7 +186,7 @@ const ButtonToggleWithForwardRef = forwardRef<HTMLButtonElement, Props>(function
   const isDarkModeRed = isDarkMode && color === 'red';
   const { isFocusVisible } = useFocusVisible();
 
-  const buttonToggleAnimation = useButtonToggleAnimation();
+  const buttonToggleAnimation = useTapScaleAnimation();
```

Comment:
is this expected?

Reply:
just renaming i guess

Reply:
yes, i renamed it cuz now they is shared